packages/mdl-ripple/index.js

```diff
@@ -32,6 +32,23 @@ export default class MDLRipple extends MDLComponent {
     return ripple;
   }
 
+  static createAdapter(instance) {
+    return {
+      browserSupportsCssVars: () => supportsCssVariables(window),
```

Comment:
Since we (pretty much) can't make any assumptions about our runtime environment within the foundation, the adapter needs a method that lets it feature-detect weather or not the browser supports the correct functionality. Furthermore, clients - especially ones that use libraries / frameworks - can use their own feature detection library if need be. Also we explicitly put this into an exported function within `./utils` so clients can just reuse ours if so desired, and I believe we actually suggest using it within our README.